### ISSUE TITLE
[ERROR] import: java.lang.NullPointerException: null

### ISSUE THREAD

**Issue by vivekdhiman** [shsteimer/IntelliVault#11]
I am getting an error while import the CRX content to intelliJIdea, and not able to debug.

**Comment by futzilogik** [comment]
I got a NullPointerException for both push and pull to / from CRX with the latest version (0.9.5) on Ubuntu Linux. I got it working on my machine by changing the working directory to the directory where IntelliJ is actually installed when starting it. Before that I used the IntelliJ launcher from my desktop environment which sets the home directory as the working directory, which didn't work. Apparently, the working directory of the IntelliJ process makes a difference to IntelliVault...

**Comment by vivekdhiman** [comment]
@padolphs  Can you please explain what changes you made briefly to make it work. I am also on ubuntu. If run the IDEA with the command line as $ /home/vivekd/softwares/idea-IC-141.1532.4/bin/idea.sh its work but with the desktop entry its doesn't.

**Comment by Faresis** [comment]
Yes, @padolphs right. To make the desktop launcher work following steps can be taken:

`sudo vi /usr/share/applications/jetbrains-idea-ce.desktop`

Change Exec line to the following:

`Exec=bash -c 'cd /opt/idea/bin && ./idea.sh'`

Those steps did the job for me.

Ubuntu 14.04 x64
IntelliJ IDEA 2016.2.4
Build #IC-162.2032.8, built on September 9, 2016
JRE: 1.8.0_112-release-b343 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o

CC: @vivekdhiman

**Comment by jrnutt** [comment]
This appears to duplicate issue #8, has there been any progress on either? I can use the work around listed above, but it's a mild nuisance.

**Comment by caffr** [comment]
Note: To fix the issue on Windows (Windows 10 in my case) - edit properties of the IntelliJ desktop shortcut and add the path to the IntelliJ 'bin' directory into 'Start in:' field. In my case it's:  "C:\Program Files\JetBrains\IntelliJ IDEA 2017.1\bin".

**Comment by felu** [comment]
Same Issue here on Windows. This is very annoying, especially because the error message is not giving any decent information. Please fix this. Thanks.

**Comment by blakesong-ecommbi** [comment]
Thanks, @caffr, It works for me

**Comment by uiblox** [comment]
@caffr Life Saver! Thank you. This resolved the NullPointerException.

**Comment by nani118066** [comment]
Any suggestions for Mac?

IntelliJ Version: 2020.2.3 Ultimate
IntelliVault: 0.10.0
Vault Cli: 3.4.2
